include/swift/Frontend/FrontendOptions.h

```diff
@@ -478,6 +478,10 @@ class FrontendOptions {
   /// crashes the compiler.
   bool emptyABIDescriptor = false;
 
+  /// Augment modular imports in any emitted ObjC headers with equivalent
+  /// textual imports
+  bool EmitObjCHeaderWithNonModularIncludes = false;
```

Comment:
You should also rename to `EmitClangHeader...` this now that you've renamed the option.